content/java/developer-guide/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Developer Guide"
+date: 2017-07-13T22:52:55-04:00
+draft: false
+mainpage: false
+projectpage: true
+projectslug: "java"
+projectname: "WorldWind Java"
+---
+
+## Developer Guide
```

Comment:
goworldwind.org's nested Symbology guides were awkward to begin with, and are awkward in the new site. Changing symbology guides requires that you navigate back to the top level Developer Guide first. What if these were tutorials instead?